readme.md

```diff
@@ -220,6 +220,23 @@ But what if your `script.js` file is not only loaded by `card.blade.php`, but al
 
 That's where this package comes to the rescue. It will load the asset just ONCE, even if it's loaded from multiple blade files.
 
+## FAQ
+
+#### Basset is not working, what may be wrong?
+
+Before making any changes, you can run the command `php artisan basset:check`. It will perform a basic test to initialize, write, and read an asset, giving you better insights into any errors.
+
+The most common reasons for Basset to fail are:
+
+1) **Incorrect APP_URL in the `.env` file.**  
+Ensure that APP_URL in your `.env` matches your server configuration, including the hostname, protocol, and port number. Incorrect settings can lead to asset loading issues.
+
+2) **Improperly configured disk.**  
+By default, Basset uses the public disk since its files must be public. For new Laravel projects, the configuration is usually correct. If you're upgrading from an older version, check `config/backpack/basset.php`. Make sure your `config/filesystems.php` public disk looks like: https://github.com/laravel/laravel/blob/10.x/config/filesystems.php#L39-L45.
```

Comment:
```suggestion
By default, Basset uses the Laravel `public` disk. 
For new Laravel projects, the configuration is usually correct. 
If you're upgrading a project and/or changed the `public` disk configuration, it's advised that you change the basset disk in `config/backpack/basset.php` to `basset`. `basset` disk is a copy of the original Laravel `public` with working configurations.
```

I think we should mention here that changing the disk is the solution if you have changed configurations in your public disk.

Reply:
That is not a Laravel docs link. That's a file in Laravel repository that the code can change and break the line links you are creating here. There is no guarantee that this L39-L45 will show the information you want to show to the users.

This is not a good solution. If you want to show the "example public disk" from Laravel, copy the disk code from that file and show it here.

Reply:
It's on a branch @pxpm, `10.x`, it will not change until the next version (if it changes it will be non breaking for sure) 🤷‍♂️
Do you think we really should remove it?

Reply:
Nop, that's not right. We don't need a BC for those lines to change. 

If Laravel decides to add another disk, or decides to add any other config before those lines, they will not match. 

You can try it. Add another disk or any config before the public and check where your links are going to point. 

I talk from experience, I can't count how many times I've followed github line links just to have to scroll up or down because they point to completely non-sense. 

This is my opinion only. We have a simple rule, please fill the following form:

@pxpm - Remove [x] | Keep [ ]
@promatik  - Remove [ ] | Keep [ ]
@tabacitu  - Remove [ ] | Keep [ ]

🤣 🚀

Reply:
It's true, changing the code will make the link stop to work ... And it will demand us to keep updating the link ... Now I have some doubts about keeping it

Reply:
You are not from Benfica, but you are a nice guy. Deep inside you, you are a nice guy! ❤️